src/amo/actions/reviews.js

```diff
@@ -77,12 +78,14 @@ export type UserReviewType = {|
 
 export function createInternalReview(
   review: ExternalReviewType | ExternalReviewReplyType,
+  lang: string = '',
 ): UserReviewType {
+  const effectiveLang = lang || 'en-US';
```

Comment:
tests in [amo/reducers/test_reviews.js](https://github.com/mozilla/addons-frontend/blob/master/tests/unit/amo/reducers/test_reviews.js) fail invariant checks without this line.

analogous calls to `selectLocalizedContent` are made by functions such as [createInternalAddonWithLang](https://github.com/mozilla/addons-frontend/blob/e2b810b382cee2254f5ba7129f9a85254e817ad3/tests/unit/amo/reducers/test_addons.js#L75C54-L75C54) that call `createInternalAddon` with `lang` set to `DEFAULT_LANG_IN_TESTS`.

Reply:
I tried a lot of different approaches with `createInternalReview` changed to `createInternalReviewWithLang`. There are many different test files which use `createInternalReview`, and the number of invariant checks failing substantially increase.